src/editor/utils.js

```diff
@@ -235,14 +237,25 @@ export const getPlacementHelpMessage = (
 				'newspack-popups'
 			);
 		case 'inline':
-			return sprintf(
-				// Translators: Trigger percentage.
-				__(
-					'The prompt will be automatically inserted about %s into article content.',
-					'newspack-popups'
-				),
-				triggerPercentage + '%'
-			);
+			return triggerCount
```

Comment:
This is not an accurate condition. When switching between percentage-insertion and block-count-insertion, `triggerCount` will stay the same and so the help text will not be updated:

<img width="274" alt="image" src="https://user-images.githubusercontent.com/7383192/151381129-73ebaf91-d617-443e-b22f-7b0f32db53f4.png">

The condition should be based on the `trigger_type` meta.